.github/workflows/ci.yml

```diff
@@ -7,6 +7,21 @@ on:
     pull_request:
 
 jobs:
+    LintRepo:
+        name: Lint Repo
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 16
+                  cache: 'yarn'
+            - run: |
+                yarn install
```

Comment:
I think ideally this action can run very quickly, and fail early (to prevent other jobs from running).